### DISCUSSION TITLE
Preview styles?

### DISCUSSION THREAD

**Discussion by tlebryk** [elvirbrk/NoteHighlight2016#154] [Ideas]
Thanks for this! Is there a way you could preview the styles in OneNote, on a Readme or somewhere in documentation? It is a bit tedious to copy and paste or look up every style until you find one that fits, especially against a given background color or OneNote background theme. Thanks again!

**Comment by JOE898** [comment]
Do you find a beautiful style?

**Reply by Knoxzen** [reply]
I liked the navy, oxygenated, neon these are some good ones
